index.html

```diff
@@ -40,4 +41,143 @@ <h1 class="heading">
+        <button type="button" class="btn-one">See project</button>
+      </div>
+      <div class="project-card">
+
+        <img src="images/photo2.png" alt="Second Card">
+        <h2 class="head">Multi-post Stories</h2>
+        <div class="card-detail-box">
+          <ul>
+            <li class="card-detail canopy">CANOPY</li>
+            <li class="card-detail option"><img src="images/bullets.png" alt="bullet"> Back End Dev</li>
+            <li class="card-detail option"><img src="images/bullets.png" alt="bullet"> 2015</li>
+          </ul>
+
+          <p> A daily selection of privtely personalized reads; no accounts or sign-ups required. </p>
+          <div class="language1 inline">
+            <ul>
+              <li class="langu">html</li>
+              <li class="langu">css</li>
+              <li class="langu">javascript</li>
+            </ul>
+          </div>
+        </div>
+        <button type="button" class="btn-one">See project</button>
+      </div>
+      <div class="project-card">
+
+        <img src="images/photo3.png" alt="Third Card">
+        <h2 class="head">Tonic</h2>
+        <div class="card-detail-box">
+          <ul>
+            <li class="card-detail canopy">CANOPY</li>
+            <li class="card-detail option"><img src="images/bullets.png" alt="bullet"> Back End Dev</li>
+            <li class="card-detail option"><img src="images/bullets.png" alt="bullet"> 2015</li>
+          </ul>
+
+          <p> A daily selection of privtely personalized reads; no accounts or sign-ups required. </p>
+          <div class="language1 inline">
+            <ul>
+              <li class="langu">html</li>
+              <li class="langu">css</li>
+              <li class="langu">javascript</li>
+            </ul>
+          </div>
+        </div>
+        <button type="button" class="btn-one">See project</button>
+      </div>
+      <div class="project-card">
+        <img src="images/photo4.png" alt="Forth Card">
+        <h2 class="head">Multi-post Stories</h2>
+        <div class="card-detail-box">
+          <ul>
+            <li class="card-detail canopy">CANOPY</li>
+            <li class="card-detail option"><img src="images/bullets.png" alt="bullet"> Back End Dev</li>
+            <li class="card-detail option"><img src="images/bullets.png" alt="bullet"> 2015</li>
+          </ul>
+          <p> A daily selection of privtely personalized reads; no accounts or sign-ups required. </p>
+          <div class="language1 inline">
+            <ul>
+              <li class="langu">html</li>
+              <li class="langu">css</li>
+              <li class="langu">javascript</li>
+            </ul>
+          </div>
+        </div>
+        <button type="button" class="btn-one">See project</button>
+      </div>
+    </section>
+    <section class="section-about">
+        <div class="about-grid">
+          <div class="text">
+            <div class="headline-text">
+              <h1>About Myself</h1>
+
```

Comment:
- [ ] After all the projects, the background color changes, and the top right corner of the about section is curved. It would be good to address this issue so that your design can look more like the Figma design.
- [ ] Also your about heading `About Myself` has no space to the left which is not the case with the Figma design. Do well to add either a margin-left or padding-left to the heading.
- [ ] The heading is also supposed to be on 2 lines but yours is only on a single line and your font size is smller. It would be nice to correct that.
- Your design:
![Capture10](https://user-images.githubusercontent.com/85113567/153602303-67f94487-9835-4b16-839c-d0e029957367.PNG)
- Figma design:
![Capture9](https://user-images.githubusercontent.com/85113567/153602348-503673f2-50f1-412b-984f-c44913f134a3.PNG)

Reply:
👍🏽 All good now.